<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_18">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #fa8137;
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: grey;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_10">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #fa8137;
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}
QPushButton::pressed{
	background-color:white;
	color:#fa8137;
}


</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: grey;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_9">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #3d3d3c;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_12">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #3d3d3c;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_20">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #3d3d3c;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_16">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #3d3d3c;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #3d3d3c;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_19">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #3d3d3c;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:146px;
 max-height:68px;
 min-width:146px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #3d3d3c;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #fa8137;
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}
QPushButton::pressed{
	background-color:white;
	color:#fa8137;
}

</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #fa8137;
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}
QPushButton::pressed{
	background-color:white;
	color:#fa8137;
}


</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_8">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #3d3d3c;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_13">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #3d3d3c;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_15">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #3d3d3c;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_14">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #fa8137;
 border-style: solid;
 border-width:1px;
 border-radius:35px;
 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}
QPushButton::pressed{
	background-color:white;
	color:#fa8137;
}

</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_11">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #3d3d3c;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: grey;
 border-style: solid;
 border-width:1px;
 border-radius:35px;

 max-width:68px;
 max-height:68px;
 min-width:68px;
 min-height:68px;
}

</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLineEdit" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>American Typewriter</family>
         <pointsize>30</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
